schemas/tsp/booking-option.json

```diff
@@ -9,15 +9,18 @@
     },
     "leg": {
       "$ref": "#/definitions/leg"
-     },
-     "terms": {
-       "$ref": "../core/booking.json#/definitions/terms"
+    },
+    "terms": {
+      "$ref": "../core/booking.json#/definitions/terms"
     },
     "meta": {
       "$ref": "../core/booking.json#/definitions/meta"
     },
     "tspProduct": {
       "$ref": "#/definitions/tspProduct"
+    },
+    "configurator": {
```

Comment:
As we already agreed to add `additional` aside of options. Wouldn't it be better to define choice configuration somewhere at `additional` and at the booking just keep list of _applicable_ config options (array of ids)  and _selection_ ?

Reply:
The configurator is supposed to be option specific, and i remember additional was something we decide to be a common piece of data for every option right?

Reply:
> The configurator is supposed to be option specific, 

From listed use cases, it looks as if config choices are not option specific.
e.g. I can imagine baby seat option to be applicable to few car options in case of rental, or many taxi choices (assuming it's taxi ride, and we list more than one taxi option).

Still, it indeed can get option specific if cost of a _config choice_ is influenced by an option (e.g. baby seat for car A will be 15€, while for car B 20€).
Or if we talk about discount, and require _flat rate_. e.g. 50% off, will have different flat rate depending on cost as set in option.

Anyway, it's just how I see it from description. I don't have full picture on how it's intended to e used

Reply:
For Rail ticket, i imagine each booking option will represent a different train, so every configuration would be different. I'd imagine the same principal applying to other TSP